automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/library/DataLibraryPage.scala

```diff
@@ -117,11 +117,11 @@ class ResearchPurposeModal(implicit webDriver: WebDriver) extends OKCancelModal(
       Thread sleep 100
       find(cssSelector(ontologySearch.query.queryString)).get.underlying.sendKeys(_)
     }
+    await condition ( invisibleSpinner, 10 )
```

Comment:
prevents test from failing

https://fc-jenkins.dsp-techops.broadinstitute.org/job/firecloud-ui-fiab-test-runner/3162/
https://fc-jenkins.dsp-techops.broadinstitute.org/job/firecloud-ui-fiab-test-runner/3166/

Reply:
Ideally, we would search for an indication that the table is ready. Otherwise if the test misses the spinner it would wait the maximum time specified.